src/sonic-yang-models/yang-models/sonic-bgp-allowed-prefix.yang

```diff
@@ -96,11 +96,13 @@ module sonic-bgp-allowed-prefix {
 
                 leaf-list prefixes_v4 {
                     type bgp-allowed-ipv4-prefix;
+                    ordered-by user;
```

Comment:
>ordered-by user; [](http://example.com/codeflow?start=20&length=16)

This PR seems include too many topics than the title mentioned. For example "ordered-by" is not explained. If it is not related to libyang v3 upgrade, we may consider a separate PR for it.